libs/datasets/sources/can_scraper_helpers.py

```diff
@@ -136,10 +135,6 @@ def _make_indexed_df(all_df: pd.DataFrame) -> pd.DataFrame:
             .rename(PdFields.DEMOGRAPHIC_BUCKET)
         )
         CanScraperLoader._check_location_id(all_df)
-        # Hack fix for bad location_ids.
-        all_df[Fields.LOCATION_ID] = _fips_from_int(all_df[Fields.LOCATION]).apply(
-            pipeline.fips_to_location_id
```

Comment:
Is this a function that can be deleted now?

Reply:
No, it is used in _check_location_id. "Trust, but verify" :-P